lib/inferno/dsl/fhir_resource_validation.rb

```diff
@@ -157,7 +157,7 @@ def exclude_message(&block)
         end
 
         # @see Inferno::DSL::FHIRResourceValidation#resource_is_valid?
-        def resource_is_valid?(resource, profile_url, runnable)
+        def resource_is_valid?(resource, profile_url, runnable, log_messages: true) # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
I don't think `log_messages` accurately describes this functionality. I think something along the lines of `persist_messages`, `add_messages_to_runnable`, or `omit_messages` would be a more accurate description of the functionality.